tests/TestCase/Validation/ValidationTest.php

```diff
@@ -1439,7 +1461,6 @@ public function testDateY()
         $this->assertFalse(Validation::date('20009', ['y']));
         $this->assertFalse(Validation::date(' 2012', ['y']));
         $this->assertFalse(Validation::date('3000', ['y']));
-        $this->assertFalse(Validation::date('1799', ['y']));
```

Comment:
Should we replace this with an equivalent (although weird) somewhere? 31-12-0000

Is simply '0' or '0000' tested somewhere?

Reply:
The year 0 is currently not supported. From what I could understand from the gregorian calendar wikipedia's pages there isn't a year 0.

Reply:
I meant technically that's the first invalid date.